onnxruntime/python/tools/kernel_explorer/kernels/gemm_fast_gelu_test.py

```diff
@@ -0,0 +1,155 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+
+import re
+import sys
+from itertools import product
+
+import kernel_explorer as ke
+import numpy as np
+import pytest
+from utils import get_basic_size, get_bert_sizes, get_gemm_bound, transab_to_suffix
```

Comment:
When the code is moved, please check the name to make sure it is still make sense in the new context. `get_basic_size`, `get_bert_sizes` is confusing the readers of what size it refers to. You should call them like `get_gemm_basic_sizes` and `get_gemm_bert_sizes`. Or if you call the file as `gemm_utils.py` then you can use `gemm_utils.get_basic_size` to avoid the confusion here.

Reply:
This confusion happens especially when the reader is reading `utils.py`